Kuroba/app/src/main/java/com/github/adamantcheese/chan/utils/StringUtils.java

```diff
@@ -90,4 +90,31 @@ public static String decodeBase64(String base64Encoded) {
 
         return bytesToHex(bytes);
     }
+
+    public static String maskPostNo(int postNo) {
+        String postNoString = String.valueOf(postNo);
+        if (postNoString.length() >= 4) {
+            return postNoString.substring(0, postNoString.length() - 3) + "XXX";
+        }
+
+        return postNoString;
+    }
+
+    public static String maskUrl(@NonNull String url) {
```

Comment:
This should be named maskImageUrl, because that's what it's doing; if you want to mask general URLs, you might want to use a regex based approach using `(\d+)`, and replacing the matching  group with a censored version.

Reply:
Done.